monitoring/base/rhods-rules.yaml

```diff
@@ -11,14 +11,14 @@ spec:
   - name: rhods-usage.rules
     rules:
     - record: cluster:usage:consumption:rhods:cpu:seconds:rate1h
-      expr: sum(rate(container_cpu_usage_seconds_total{container="",pod=~"jupyterhub-nb.*",namespace="rhods-notebooks"}[1h]))
+      expr: sum(rate(container_cpu_usage_seconds_total{container="",pod=~"jupyter-nb.*",namespace="rhods-notebooks"}[1h]))
     - record: cluster:usage:consumption:rhods:pod:up
-      expr: count(kube_pod_container_status_ready{namespace="rhods-notebooks",pod=~"jupyterhub-nb.*"}==1)
+      expr: count(kube_pod_container_status_ready{namespace="rhods-notebooks", pod=~"jupyter-nb.*",container=~"jupyter-nb-.*[^kube+-3aadmin]"}==1)
     - record: cluster:usage:consumption:rhods:active_users
-      expr: count(count(count_over_time(kube_pod_container_status_ready{namespace="rhods-notebooks",pod=~"jupyterhub-nb.*"}[1h])) without (uid))
+      expr: count(count(count_over_time(kube_pod_container_status_ready{namespace="rhods-notebooks",pod=~"jupyter-nb.*", container=~"jupyter-nb-.*[^kube+-3aadmin]"}[1h])) without (uid))
```

Comment:
I think we can replace this ugly expression with the runningNotebooks metric that KFNBC provides.

Reply:
Seems that the rule record cluster:usage:consumption:rhods:active_users doesn't work properly, with the count(notebook_running) expression. 

On the other hand the old expression is working fine: count(count(count_over_time(kube_pod_container_status_ready{namespace="rhods-notebooks",pod=~"jupyter-nb.*", container=~"jupyter-nb-.*"}[1h])) without (uid))

I will investigate it further

Reply:
This metric `count(kube_statefulset_replicas{namespace=~"rhods-notebooks", statefulset=~"jupyter-nb-.*"} ==1)` works better for the `cluster:usage:consumption:rhods:active_users`